src/main/java/com/vailsys/freeclimb/api/call/CallsRequester.java

```diff
@@ -115,7 +115,8 @@ public Call create(String to, String from, String applicationId) throws FreeClim
 	 *                      purchase from FreeClimb or a verified phone number owned
 	 *                      by the user.
 	 * @param applicationId The {@code applicationId} for the registered FreeClimb
-	 *                      application which should handle this call.
+	 *                      application which should handle this call. Required if
+	 *                      no `parentCallId` or `callConnectUrl` has been provided.
```

Comment:
Making a call without an applicationId being accomplished by providing a `null` applicationId and setting values in `CallOptions` feels wrong. Unfortunately, since the other two options (callConnectUrl and parentCallId) are both strings we cannot make overloading functions which expose those fields directly to the user. 

I think we need to move applicationId into `CallOptions` or something like that. Maybe extract parentCallId and callConnectUrl from CallOptions into a new type alongside applicationId? I don't know.

Reply:
I agree. When I originally did this work, I moved the applicationId into the `CallOptions`, but I wasn't sure if that change was larger than the documentation describes. Since you also agree it can be moved to `CallOptions`, that will be my course of action. Thank you for the feedback!

Reply:
After testing attempts to create calls without the Application ID, it appears it is required for all outbound call requests with FreeClimb so instead of moving the applicationId to the CallOptions, I changed the descriptions provided here in the Java SDK.

Reply:
The docs indicate that an applicationId can be omitted if a callConnectUrl is specified: https://docs.freeclimb.com/reference/calls-1#applicationid-v-callconnecturl

Reply:
Yes, I believe this documentation needs changed as well. When I try to use a callConnectUrl without applicationId, the call fails from invalid json due to a schema error. The original documentation was also inconsistent on whether the applicationId is required (requirement 3d under SDK updates says the application ID is required when creating a call). https://wiki.vailsys.com/display/VCSHOME/Add+CallConnectURL+Attribute+to+Make+a+Call+with+Query+Params#AddCallConnectURLAttributetoMakeaCallwithQueryParams-SDKUpdates

Reply:
I'm very sure that the ability to make a call without passing an applicationId is intended, so if it's not working it's not the docs/SDK that are wrong.

Reply:
Oh I see, I wasn't reading carefully enough. The applicationId is always required, it's just not required that the application being referred to have a callConnectUrl defined if one is provided in the request.